docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -566,6 +566,33 @@ Here one could find description of general HTTP properties applied across all th
 
 |===
 
+== Properties externalization
```

Comment:
```suggestion
== External test configuration points
```

Reply:
maybe "Test configuration externalization" ? "points" looks a bit weird

Reply:
"externalization" is too complicated